recipe/gen_patch_json.py

```diff
@@ -993,6 +993,28 @@ def _gen_new_index(repodata, subdir):
         ):
             _add_pybind11_abi_constraint(fn, record)
 
+        # do some fixes for pytorch, pytorch-cpu & gpu
+        if (
+            record_name == "pytorch-cpu"
+            and (
+                pkg_resources.parse_version(record["version"])
+                < pkg_resources.parse_version("1.6")
+            )):
+            record.setdefault('constrains', []).extend([
+                "pytorch-gpu 9999999999"
+            ])
+
+        if (
+            record_name in ("pytorch-cpu", "pytorch-gpu")
+            and
+                pkg_resources.parse_version(record["version"])
+                == pkg_resources.parse_version("1.6")
```

Comment:
Does this work when version is 1.6.0? Can you update the diff?

Reply:
yep, that works with this special pkg_resources object.